src/app/components/content/IsaacSymbolicChemistryQuestion.tsx

```diff
@@ -1,68 +0,0 @@
-import React, {lazy, useEffect, useMemo, useRef, useState} from "react";
-import {IsaacContentValueOrChildren} from "./IsaacContentValueOrChildren";
-import {ChemicalFormulaDTO, IsaacSymbolicChemistryQuestionDTO} from "../../../IsaacApiTypes";
```

Comment:
When removing all these files, have you also checked the imports as well? For instance, `ChemicalFormulaDTO` is no longer used after your changes, so I would imagine that means that it can also be removed.